.gitignore

```diff
@@ -0,0 +1 @@
+*.txt
```

Comment:
I notice a lot of pyc files got into your commits.  These should probably be included here.

Since they are already there they won't be removed.  You could however do something like

`find . -name "*.pyc" | xargs git rm`

This should git remove the files.

Then inclucde `**/*.pyc` in your gitignore and that will keep new ones from creeping in.

Reply:
I'm marking this as a requested change, please give it a shot and let me know if you hit any issues.  Create a new branch just to be safe and daisy chain it.  That way if it gets messy we can back up to this one.  Thanks

Reply:
@EfrainOlivaresUVA Please see pull request #10